src/CognitiveServices/CognitiveServices.sln

```diff
@@ -7,6 +7,12 @@ EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Tests", "Tests", "{13BB1C60-A70E-4780-ABEF-D5EE36754085}"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "CognitiveServices.Test", "CognitiveServices.Test\CognitiveServices.Test.csproj", "{F6F8A360-F203-4161-98A7-60097AB94193}"
+	ProjectSection(ProjectDependencies) = postProject
```

Comment:
Do you need these entries in the solution file? If you are not sure why they were added could you remove and test that everything is ok?

Reply:
Its not needed (auto added by VS)